### DISCUSSION TITLE
Power up in FM VHF mode

### DISCUSSION THREAD

**Discussion by brookbonde** [egzumer/uv-k5-firmware-custom#178] [Ideas]
Would like to see an option to default to VHF FM on start up, or at least return back to VHF FM operation if that was the last used before power off.

**Comment by ErikS-web** [comment]
Default is last used channels/vfo's.

Your wish can also be achieved quickly in an alternative way ... For this there is function.... [`9 call`](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Button-functions#9-call)

**Reply by brookbonde** [reply]
If I select FM (Using the FM button) and then turn the radio off and back on, it does not return to the FM VHF band, it goes back to the default VFO screen.  I cannot find a way to make the radio start up on VHF FM.

**Comment by brookbonde** [comment]


**Reply by ErikS-web** [reply]
Please make your own  item in Discussions. Do not extend to other subject
